### ISSUE TITLE
Missing a pair of stereo images with respect to the training of KITTI2015 dataset

### ISSUE THREAD

**Issue by wujunjie15** [feihuzhang/GANet#36]
Dear,
    when i implemented your code, i found that the image sequence of 000104_10 is not added to the KITTI2015 training set in kitti2015_train.list.
    Additionally, i pay attention to the accuracy of results if i run it on a nNvidia GeForce GTX 1080ti GPU due to the limitation of the provided experimental equipment. How many times the difference between the obtained accuracy of disparities and that of GANet-deep? Can you approximately infer it?

**Comment by feihu2020** [comment]
I found that's a "very dark" image, so directly remove it. But, this is not important. You can add it to the list for training.

I didn't run the training on a single 1080ti gpu. Could not give accurate estimation. But, I think accuracy would be bad. Since batchsize can only be one or two.

You can try to only update weights once after several iteration to approximate the performance of multi-GPU. (in this case, just replace the sync_bn with normal bn.)


I also suggest to simplify the model. For example, remove the hour-glass to use a simple encoder-decoder with few layers. Until you can use 4 as batchsize in your own GPU. 4 might be enough.
 
The code can be improved to reduce the GPU mem consumption. But, currently, sorry that I do not have much time to do this.
